features/polypolyMembership/src/polypolyMembership.jsx

```diff
@@ -1,24 +1,28 @@
 import React from "react";
 import { createRoot } from "react-dom/client";
-import { MemoryRouter as Router, Routes, Route } from "react-router-dom";
+import { MemoryRouter, Routes, Route } from "react-router-dom";
 
 import Onboarding from "./views/onboarding.jsx";
 import Overview from "./views/overview.jsx";
 
+import { MembershipContextProvider } from "./context/membership-context.jsx";
+
 import "./styles.css";
 
 const PolypolyMembership = () => {
     return (
-        <Router>
-            <div className="poly-nav-bar-separator-overlay" />
-            <div className="feature-container membership">
-                <Routes>
-                    <Route path="/" exact element={<Onboarding />} />
-                    <Route path="/onboarding" element={<Onboarding />} />
-                    <Route path="/overview" element={<Overview />} />
-                </Routes>
-            </div>
-        </Router>
+        <MemoryRouter>
+            <MembershipContextProvider>
+                <div className="poly-nav-bar-separator-overlay" />
+                <div className="feature-container membership">
+                    <Routes>
+                        <Route path="/" exact element={<Onboarding />} />
+                        <Route path="/onboarding" element={<Onboarding />} />
+                        <Route path="/overview" element={<Overview />} />
+                    </Routes>
+                </div>
+            </MembershipContextProvider>
+        </MemoryRouter>
```

Comment:
I actually think we might want to change to BrowserRouter at some point and could start here. Going back in the Browser could then be used to go back in the app

Reply:
There's a non-zero chance that something related to that might be causing the errors. See my (frustratingly long) thread in Slack

Reply:
Could you point me to the thread?

Reply:
https://polypoly-team.slack.com/archives/C03PPA04UDC/p1664171029490069